test/test-fixtures/transformations/resolved html/p-applicant-british-citizen-or-eu-national.js

```diff
@@ -134,13 +134,20 @@ const html = `<!DOCTYPE html>
   </p>
 </div>
 
-    <div class="govuk-width-container">
-
-
+<div class="govuk-width-container">
+        
+        <div class="govuk-grid-column-two-thirds">
+            
+                
                 <a href="/apply/previous/applicant-british-citizen-or-eu-national" class="govuk-back-link">Back</a>
 
-
-    </div>
+            
+        </div>
+        <div class="govuk-grid-column-one-third">
+            
+        </div>
+            
+  </div>
 
```

Comment:
I know this is just a est fixture, but this seems very unity? Why do we have an empty div now?

Reply:
It is to test the absence of the link. You are making a good point about the div itself not being a useful HTML element in this context. It is the nature of this test fixture. It isn't great.

If it makes more sense (don't see why it wouldn't) If can move the if statement to wrap the div element in the form-helper.js and that would avoid the need to have the empty div in the test.